vss_invoice_mgmt/src/pages/business/pages/masters/vendormaster/pages/VendorMaster.jsx

```diff
@@ -0,0 +1,337 @@
+import { useFormik } from "formik";
+import Sidebar from "@/components/Sidebar/Sidebar";
+import HeaderStats from "@/components/Headers/HeaderStats";
+import FooterAdmin from "@/components/Footers/FooterAdmin";
+import AdminNavbar from "@/components/Navbars/AdminNavbar";
+import { Button, TextField } from "@mui/material";
+import { Link, useParams } from "react-router-dom";
+import React, { useState, useEffect } from "react";
+import * as Yup from "yup";
+import Vendor_Statedropdown from "@/components/Vendor-State/Vendor_Statedropdown";
+import StatusSelect from "@/components/Status-Select/Status_Select";
+import { ToastContainer, toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+import {
+  PostVendorMasterList,
+  GetVendorMasterById,
+  PutVendorMasterList,
+} from "../vendormasterconfig.js";
+import Textarea from "@mui/joy/Textarea";
+
+const vendormaster_ValidationSchema = Yup.object().shape({
+  vendorName: Yup.string()
+    .required("Please enter vendor master name")
+    .min(3, "Vendor master name must be at least 3 characters")
```

Comment:
Correct the validation messages

Reply:
I have kept the message according to your product category code.